### DISCUSSION TITLE
over valve / window detection

### DISCUSSION THREAD

**Discussion by Lux-WorldPC** [jmcollin78/versatile_thermostat#577] [General]
I noticed that when the windows is open, vtherm switches to eco like requested, but if scheduler sets to confort, it goes to confort and opens de valve if needed while windows open is still detected....
<img width="652" alt="Screenshot 2024-10-23 at 06 54 19" src="https://github.com/user-attachments/assets/4a038a28-2854-473b-993c-8ad657239025">
<img width="430" alt="Screenshot 2024-10-23 at 06 53 59" src="https://github.com/user-attachments/assets/e987338c-f4ce-47de-882c-94eb3562f6cf">

**Comment by jmcollin78** [comment]
Hello, yes this a normal behavior to allow someone to force heating when window is dectected.

You should rather set to off if window is detected, so the Scheduler will set the preset to comfort but the VTherm is still off. When the window is closed, then the VTherm heats to comfort.
